deeplearning4j/deeplearning4j-nlp-parent/deeplearning4j-nlp/src/main/java/org/deeplearning4j/models/embeddings/loader/WordVectorSerializer.java

```diff
@@ -2672,11 +2672,20 @@ protected static TokenizerFactory getTokenizerFactory(VectorsConfiguration confi
      * @param file File should point to previously saved w2v model
      * @return
      */
+    public static WordVectors loadStaticModel(InputStream inputStream) throws IOException {
+
+        File tmpFile = DL4JFileUtils.createTempFile("word2vec"+System.currentTimeMillis(), "tmp");
+        tmpFile.deleteOnExit();
+        FileUtils.copyInputStreamToFile(inputStream, tmpFile);
+        return loadStaticModel(tmpFile);
```

Comment:
Better IMO to use:
```
try{
    return loadStaticModel(tmpFile);
} finally {
   tmpFile.delete();
}
```
This is better for long-lived JVMs (and those that don't exit properly).